m4opt/_cli/schedule.py

```diff
@@ -275,7 +282,20 @@ def schedule(
         observer_locations = mission.observer_location(obstimes)
 
     with status("evaluating field of regard"):
-        target_coords = mission.skygrid
+        if isinstance(mission.skygrid, dict):
+            if skygrid is not None:
+                if skygrid not in mission.skygrid:
+                    typer.secho(
+                        f"skygrid '{skygrid}' not found. Options: {', '.join(mission.skygrid.keys())}",
+                        fg=typer.colors.RED,
+                    )
+                    raise typer.Exit(1)
+                target_coords = mission.skygrid[skygrid]
+            else:
+                target_coords = next(iter(mission.skygrid.values()))
```

Comment:
Maybe it would be better to have no default, and error out if there is more than one sky grid and none are specified.